<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QDialog" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="oldPasswordLE">
       <property name="placeholderText">
        <string>old password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newPasswordLE">
       <property name="placeholderText">
        <string>new password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newPasswordConfirmLE">
       <property name="placeholderText">
        <string>confirm new password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changePasswordPB">
       <property name="text">
        <string>Change Password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
